packages/components/src/lib/controllers/ui-controller.js

```diff
@@ -85,20 +96,20 @@ export default class UIController {
     this.TAB_NAMES_DISABLED = 'disabled'
   }
 
-  init ({ api, store, uiOptions } = {}) {
+  init ({ api, store } = {}) {
     if (!api) {
       throw new Error('API object is required for a UI controller initialization')
     }
     if (!store) {
       throw new Error('Vuex store is required for a UI controller initialization')
     }
-    if (!uiOptions) {
-      throw new Error('UI options are required for a UI controller initialization')
-    }
 
     this._api = api
     this._store = store
-    this._uiOptions = uiOptions
+    this._uiOptions = this._api.settings.getUiOptions()
+
+    // Inject HTML code of a plugin. Should go in reverse order.
+    document.body.classList.add(injectionClasses.ALPHEIOS)
```

Comment:
why do we do this here instead of in activateOnPage?

Reply:
`activateOnPage()` does changes revert changes that were made after deactivation. It removes CSS classes that disables text selection. 
The code here, however, does what needs to be done only once, during an app initialization: an addition of an `alpheios` class the the document's body.
That's the difference.

Reply:
ok thanks.